cert-generation/config/constants.go

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package config
+
+import "k8s.io/api/admissionregistration/v1beta1"
+
+// Constants for webhook
+const (
+	MutateWebhookName          = "mutate.che-workspace-controller.svc"
+	MutateWebhookCfgName       = "controller.devfile.io-blocker"
```

Comment:
When I see the code I confuse that with constant from controller. It's better to name Blocker and usual WebhookCfg differently